src/Misc/XMLwrapper.cpp

```diff
@@ -92,16 +121,22 @@ XMLwrapper::XMLwrapper()
     minimal = true;
     SaveFullXml=false;
 
-    node = tree = mxmlNewElement(MXML_NO_PARENT,
-                                 "?xml version=\"1.0f\" encoding=\"UTF-8\"?");
```

Comment:
aha, that explains the origin of the 1.0f, it must have gotten changed when we were clarifying float64 vs float32 types in the past.

Reply:
Indeed: d44dc9b66d503405cac29b73cdd15c962f0098b2 changed that in 2011(!), likely with the use of a recursive `sed` call :laughing: